fractions/jose/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2015 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.wildfly.swarm</groupId>
+    <artifactId>build-parent</artifactId>
+    <version>2018.6.0-SNAPSHOT</version>
+    <relativePath>../../build-parent/pom.xml</relativePath>
+  </parent>
+
+  <groupId>org.wildfly.swarm</groupId>
+  <artifactId>jose</artifactId>
+
+  <name>JOSE Signing and Encryption Support</name>
+  <description>Supports JWS signing and JWE encryption</description>
+
+  <packaging>jar</packaging>
+
+  <properties>
+    <swarm.fraction.stability>experimental</swarm.fraction.stability>
+    <swarm.fraction.tags>Signing,Encryption</swarm.fraction.tags>
+    <version.cxf>3.2.5-SNAPSHOT</version.cxf>
```

Comment:
Shouldn't be merged with a SNAPSHOT dependency I think.

Reply:
Sure, you are right, the release is expected shortly, I'll update once it is there. I only wonder how to get it to the new repo without delaying the migration. May be remove the links to the fraction/tests from the root pom while SNAPSHOT is there...